### ISSUE TITLE
Additional help resources

### ISSUE THREAD

**Issue by sudo-may** [NationalSecurityAgency/skills-service#3352]
In the `Help` Menu on the top-right: 
- add link to `Training Participation Guide`, name the link `Training`
- change `Dashboard` link to `Admin`
- add link to `Accessibility Guide`, the link should be page-context aware and properly select between the Training vs. Admin version of the guide; name the link `Accessibility`
